### PR TITLE
Update ru_ru.json (this one is for grab-mechanic branch)

### PR DESCRIPTION
**Pull request by spacedan228**
add changes from master to grab mechanic branch pls

### CODE REVIEW

File: src/main/resources/assets/changed/lang/ru_ru.json [hunk 1]
```diff
@@ -509,9 +515,14 @@
   "eyestyle.changed.v2": "Широкий",
   "eyestyle.changed.tall": "Узкий",
 
+ "changed.config.color_picker_red": "Красный: %s",
+ "changed.config.color_picker_green": "Зеленый: %s",
+ "changed.config.color_picker_blue": "Синий: %s",
```
Comment (LtxProgrammer) [inline src/main/resources/assets/changed/lang/ru_ru.json:520]: These will be duplicated when I merge the branches

File: src/main/resources/assets/changed/lang/ru_ru.json [hunk 2]
```diff
@@ -509,9 +515,14 @@
...
 
+ "changed.config.color_picker_red": "Красный: %s",
+ "changed.config.color_picker_green": "Зеленый: %s",
+ "changed.config.color_picker_blue": "Синий: %s",
+
   "changed.config.bpi.screen": "Основная информация о игроке",
   "changed.config.bpi.hair_color": "Цвет волос",
-  "changed.config.bpi.iris_color": "Цвет радужки",
+  "changed.config.bpi.iris_color.left": "Цвет левой радужки",
+  "changed.config.bpi.iris_color.right": "Цвет правой радужки",
```
Comment (LtxProgrammer) [inline src/main/resources/assets/changed/lang/ru_ru.json:525]: And so will these

**Review by LtxProgrammer — CHANGES_REQUESTED** [review]
When I merge master into grab-mechanic with both of your PR's there will be duplicate language entries. You'll need to remove them from this PR.

**Comment by spacedan228** [comment]
removed

**Review by LtxProgrammer — APPROVED** [review]

**Pull request merged**